### PR TITLE
TW-254: add function share temple analytics on click

### PR DESCRIPTION
**Pull request by letier3110**

### CODE REVIEW

**Review by LendiHop — APPROVED** [review]

File: src/screens/settings/settings.tsx [hunk 1]
```diff
@@ -119,6 +123,20 @@ export const Settings = () => {
...
+            <WhiteContainerAction
+              onPress={() => {
+                trackEvent(SettingsSelectors.Share, AnalyticsEventCategory.ButtonPress);
+                Share.share({
+                  message:
+                    'Hey friend! You should download Temple and discover the Tezos world with me https://templewallet.com/mobile',
+                  url: 'https://templewallet.com/mobile'
+                });
+              }}
+            >
```
Comment (unixvb) [inline src/screens/settings/settings.tsx:136]: move into variable

File: src/screens/settings/settings.tsx [hunk 2]
```diff
@@ -119,6 +123,20 @@ export const Settings = () => {
               <WhiteContainerText text="About" />
               <Icon name={IconNameEnum.ChevronRight} size={formatSize(24)} />
             </WhiteContainerAction>
+            <WhiteContainerDivider />
+            <WhiteContainerAction
+              onPress={() => {
+                trackEvent(SettingsSelectors.Share, AnalyticsEventCategory.ButtonPress);
+                Share.share({
```
Comment (unixvb) [inline src/screens/settings/settings.tsx:130]: if `Share` returns `Promise` try to send into analytics successful share & share rejection

Comment (letier3110) [reply src/screens/settings/settings.tsx:130]: ok, good point

**Review by LendiHop — DISMISSED** [review]

File: src/screens/settings/settings.tsx [hunk 3]
```diff
@@ -119,6 +127,25 @@ export const Settings = () => {
               <WhiteContainerText text="About" />
               <Icon name={IconNameEnum.ChevronRight} size={formatSize(24)} />
             </WhiteContainerAction>
+            <WhiteContainerDivider />
+            <WhiteContainerAction
+              onPress={() => {
```
Comment (unixvb) [inline src/screens/settings/settings.tsx:132]: I mean to move opPress handler into variable

**Review by unixvb — APPROVED** [review]

**Review by unixvb — APPROVED** [review]

**Review by LendiHop — APPROVED** [review]

**Review by oryhivskiy — APPROVED** [review]

**Pull request merged**